element_animal/subject.py

```diff
@@ -126,9 +138,14 @@ class Lab(dj.Part):
         -> master
         -> Lab
         ---
-        subject_alias=''    : varchar(32)  # alias of the subject in this lab, if different from the id
+        subject_alias='' : varchar(32) # alias for lab, if different from id
         """
 
+    @classmethod
+    def make_nwb(cls, subject_key):
+        from .export import subject_to_nwb
+        return subject_to_nwb(subject_key)
+
```

Comment:
Hi @CBroz1 @ttngu207, why was this added as a method here?  Thanks.

Reply:
@kabilar, I don't fully understand the question here, how are we intending for users to use this `subject_to_nwb` ?

Reply:
Do you also mean why does it have to be a `classmethod`?

Reply:
Thanks @ttngu207.  Yes, exactly, how would users use `make_nwb` and `subject_to_nwb`?  I presume we would use the export function by running `subject_to_nwb({'subject':<varchar(8)>})`.  So I am not sure what `make_nwb` does?

Reply:
After discussion with @ttngu207, this method was intended for the table to be able to export its data with the `Subject.make_nwb()` command instead of `element_animal.export.nwb import subject_to_nwb`.  Since the `element-array-ephys` NWB export function does not seem to be implementing this strategy, let's remove for now to be consistent.

Reply:
Gotcha, all set. Thanks for thinking this through